src/diepvries/table.py

```diff
@@ -2,18 +2,78 @@
 
 import logging
 from abc import ABC, abstractmethod
+from datetime import datetime
 from functools import cached_property
 from typing import Dict, List
 
 from . import HASH_DELIMITER, METADATA_FIELDS, FieldRole, FixedPrefixLoggerAdapter
 from .field import Field
-from .template_sql.sql_formulas import HASHKEY_SQL_TEMPLATE
+from .template_sql.sql_formulas import (
+    HASHKEY_SQL_TEMPLATE,
+    STAGING_PHYSICAL_NAME_SQL_TEMPLATE,
+)
 
 
 class Table(ABC):
+    """A generic table.
+
+    Abstract class Table. Holds common properties between all database tables.
+    """
+
+    # pylint: disable=too-few-public-methods
```

Comment:
I didn't know that one existed. We could also disable this check, WDYT?

Reply:
This one is fine for me tbh, at least it forces you to make a conscious decision.